docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Staging release notes
 
+## November 2023
+
+**09 November 2023**
+
+Frontend version: 1.0.0-20231102.1605 | Backend version: 1.71.2-20231108.1413
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** |  Automated self termination and termination requests by Public officers | In our [service request form](https://go.gov.sg/seed-techpass-support), users can now automate the account termination process for employees who have left the organization. Previously handled manually by L1 and L2 engineers, this process can now be automated to save time. Automation is achieved by calling our Automation API, where the following steps are executed: <br>- Retrieve user information using the user's UPN.<br>- Utilise the `Update user status` endpoint (exclusive to our automation API) to terminate the user's account. <br><br><b>Note</b>: Automation applies only to cases initiated by a Public officer or when the user terminates their account not offboarded to SEED. |
+| **Enhancement** | Removed unused fields from `List User API` response in Automation API | We have removed unused fields: `lastSignInAt`, `lastInteractiveSignInAt`, `lastNonInteractiveSignInAt`, and `seedDevices`. These fields, which were previously included in the API documentation, were found to be unused and have been removed to align with the actual response. |
+| **Enhancement** | Webhook UI enhancements |  Webhooks now display the source application (TechPass or TechBiz), providing clarity to new tenant administrators. A new indicator has been added to highlight CAM required webhook events, specifically for `user-updated` and `user-deleted events`. |
+| **Enhancement** | Removed `userType` from Automation API |  WWe have removed the `userType` field from Automation API. |
```

Comment:
Can we give a background on what the userType is and what they should use?

-- Background --
The userType is a concept in Azure AD and the values can either be Guest or Member. We've used Guest to represent public officers and Member to represent vendors for some time and it was inaccurate as there are nuances to this, some public officers are created as Member accounts and some vendors are created as Guest accounts.

Since then we've introduced the accountType to accurately differentiate public officers from vendors, as well as temps. We are dropping the userType in our payloads because the accountType was introduced months ago and TechPass tenants have successfully transitioned to use the accountType field.